ceph.spec.in

```diff
@@ -1754,7 +1759,15 @@ if [ $1 -ge 1 ] ; then
 fi
 
 %files mgr-dashboard
-%{_datadir}/ceph/mgr/dashboard
+%define dashdir %{_datadir}/ceph/mgr/dashboard
+%dir %{dashdir}
+%{dashdir}/*.py
+%{dashdir}/controllers/*.py
+%{dashdir}/plugins/*.py
+%{dashdir}/services/*.py
+%{dashdir}/frontend/dist
+%{dashdir}/frontend/package*.json
+%doc %{dashdir}/README.rst
```

Comment:
probably we can remove the uninstalled bits in the `%install` section?

Reply:
Sure, I think that was the reason for the rpmbuild failure. Thanks!